lib/modules/manager/npm/extract/locked-versions.ts

```diff
@@ -80,6 +80,16 @@ export async function getLockedVersions(
         } else {
           packageFile.extractedConstraints!.npm = '<9';
         }
+      } else if (lockfileVersion === 3) {
+        if (!packageFile.extractedConstraints?.npm) {
+          packageFile.extractedConstraints!.npm = '>=7';
```

Comment:
There is pending PR which is related to this code: https://github.com/renovatebot/renovate/pull/22405/files

Please, change your code accordingly once it's merged